packages/components/constraints/src/index.js

```diff
@@ -1,6 +1,16 @@
 // eslint-disable-next-line import/prefer-default-export
 import Horizontal from './horizontal';
+import {
+  getMaxPropEquivalent,
+  parseHorizontalConstraintProp,
+  getAcceptedMaxPropValues,
+} from './helpers';
 
-export default { Horizontal };
+export default {
+  Horizontal,
+  getMaxPropEquivalent,
+  parseHorizontalConstraintProp,
+  getAcceptedMaxPropValues,
+};
```

Comment:
Why do we need to expose all these helpers?

Reply:
`parseHorizontalConstraintProp` will be used for components that have a `horizontalConstraint` prop which will accept both of the scales (until we deprecate the letters scale)
`getAcceptedMaxPropValues` is useful for storybook examples (and also propTypes)l
I can remove `getMaxPropEquivalent` from there as it is used by `parseHorizontalConstraintProp` and doesn't need to be exposed itself anymore.

Reply:
Ok...my main concern is "how" we expose those helpers. Given that we only need them for the "migration period", I would make it more explicit that people shouldn't use and rely on those.

Therefore, I would avoid exposing them as "normal exports". Instead we can have a new entry point that other components can use a deep import. Then it's easier to remove it without (breaking) changing the official component exports.

The new entry point can be for example a `helpers.js` file, or even a `experimental-helpers.js`.

The alternative is to have those exports but then we need to be aware that when we remove them it's going to be a breaking change.

Reply:
Ah, that's right. We shouldn't expose them externally.